api/client/client.go

```diff
@@ -382,11 +382,11 @@ func (t tokenAuth) GetRequestMetadata(ctx context.Context, _ ...string) (map[str
 	md := map[string]string{}
 	token, ok := ctx.Value(authKey("token")).(string)
 	if ok && token != "" {
-		md["Authorization"] = "Bearer " + token
+		md["authorization"] = "bearer " + token
```

Comment:
Why's this?

Reply:
same thing as the x-scope. the grpc client lib in java and javascript lowercase these no matter my input, so I wanted to ensure we matched here. can change it back if you prefer, just felt like the grpc preference for lowercase